### PR TITLE
table added for all charts generated by new json api

### PR DESCRIPTION
**Pull request by benstoltz**
@tomwayson thoughts/opinions?

### CODE REVIEW

**Comment by ajturner** [comment]
This is an interesting feature for anyone, not just Accessibility. The user can either flip to the table view or expand a table out to see both at the same time.

**Comment by benstoltz** [comment]
@ajturner really good point. I'd propose that for the first pass for OD we just do it for accessibility purposes. Then we go back through and add in options to the api that enable those other options you mentioned. 

Alternatively we could just attach an id to the table and manipulate it externally from cedar. @tomwayson what do you think?

File: src/cedar.js [hunk 1]
```diff
@@ -552,6 +560,68 @@ export default class Cedar {
...
+    const tableEl = document.createElement('table');
+    const tableHead = this._tableHead(series);
+    const tableBody = this._tableBody(data, series);
+    // Put everything together
+    tableEl.appendChild(tableHead);
+    tableEl.appendChild(tableBody);
+    // Make table invisible
+    tableEl.setAttribute('style', 'display: none');
+    // Assign table
+    parentEl.insertBefore(tableEl, vegaEl);
```
Comment (tomwayson) [inline src/cedar.js]: Can we insert after? Seems like less potential to mess things up.

**Comment by tomwayson** [comment]
Also looping in @FelFly to see if she has anything to say about the DOM this generates. Does it need aria attributes etc?

File: src/cedar.js (hunk 1 above)
Comment (benstoltz) [inline src/cedar.js]: Yeah, can add an aria-role to the `vegaEl` to get the screen readers to skip it.

**Comment by benstoltz** [comment]
Yep, need to change the `dipslay:none` per @FelFly that blocks screenreaders also.

**Comment by ajturner** [comment]
status?

**Comment by tomwayson** [comment]
@benstoltz can we close this?

**Comment by FelFly** [comment]
I don't have the bug marked as fixed yet that this particular thing was going to solve: https://esriarlington.tpondemand.com/entity/76533 Is this getting resolved through other channels?

**Comment by benstoltz** [comment]
@FelFly AmCharts handles accessibility so it will be covered with that.

**Comment by tomwayson** [comment]
closing as we have the Hub TP story to track any requirements not met by amCharts out of the box

**Pull request closed without merging**